plans/agent_summary.pp

```diff
@@ -6,8 +6,9 @@
 #   Defaults to using a PuppetDB query to identify nodes
 # @param indicator_exclusions
 #   List of disabled indicators, place any indicator ids you do not wish to report on in this list
+#   Can be used with a code manager/control repo to exclude test by ID using a static.yaml file
```

Comment:
The Phrasing here is not quite right.

You would need something along the lines of "Static Hiera data can be used to set indicator_exclusions see https://www.puppet.com/docs/pe/latest/writing_plans_in_puppet_language_pe.html#using_hiera_with_plans"